README.md

```diff
@@ -1,6 +1,17 @@
 # CSI NFS driver
 
-## Kubernetes
+## Overview
+
+This is a repository for [NFS](https://en.wikipedia.org/wiki/Network_File_System) [CSI](https://kubernetes-csi.github.io/docs/) Driver.
+Currently it implements bare minimum of the CSI and the state is alpha.
+
+#### CSI Feature matrix
+
+| **nfs.csi.k8s.io** | CSI versions compatibility| Dynamic Provisioning | Resize | Snapshots | Raw Block | AccessModes              | Status |
```

Comment:
Can you also add the minimum K8s version to this table? It is 1.14 due to the dependency on CSIDriver beta object.